--- CMakeLists.txt.orig   2025-07-26 17:51:29.217594626 +0000
+++ CMakeLists.txt        2025-07-26 17:51:35.989691214 +0000
@@ -48,8 +48,8 @@
 include(enable_sanitizer)
 
 # Checks : -------------------------------------------------------------------------------------------------------
-if (NOT "${CMAKE_CXX_COMPILER_ID}" MATCHES "Clang")
-    message(FATAL_ERROR "Only Clang is supported!")
+if (NOT "${CMAKE_CXX_COMPILER_ID}" MATCHES "Clang|GNU")
+    message(FATAL_ERROR "Only Clang and GCC are supported!")
 endif ()
 if (CMAKE_CXX_COMPILER_VERSION VERSION_LESS 13)
     message(FATAL_ERROR "Only Clang >= 13 is supported!")
